Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,554 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+using System.Linq;
+using Amazon.S3;
+using Amazon;
+using Amazon.S3.Model;
+
+namespace Frends.AmazonS3.DownloadObject.Tests
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        private readonly string? _accessKey = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_AccessKey");
+        private readonly string? _secretAccessKey = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_SecretAccessKey");
+        private readonly string? _bucketName = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_BucketName");
+        private readonly string _dir = Path.Combine(Environment.CurrentDirectory); // .\Frends.AmazonS3.DownloadObject\Frends.AmazonS3.DownloadObject.Test\bin\Debug\net6.0\
+
+        Connection? _connection;
+        Input? _input;
+        Options? _options;
+
+
+        /// <summary>
+        /// Create test files and folders. 
+        /// !!! Manually upload test files to S3 "2022/"-folder. You can copy files from ".\Frends.AmazonS3.DownloadObject\Frends.AmazonS3.DownloadObject.Test\bin\Debug\net6.0\DownloadTestFiles" after first test run. !!!
+        /// </summary>
+        [TestInitialize]
+        public void Initialize()
```

Comment:
You should use `OneTimeSetup` to upload the needed test files to S3. Those files should be deleted in `OneTimeTearDown`.

Reply:
Some of these files are overwritten by tests so it was easier to delete and create again to make sure they ready for the next test, and also needed less duplicate code.